### ISSUE TITLE
Dark theme with higher contrast please

### ISSUE THREAD

**Issue by afgks** [birchill/10ten-ja-reader#692]
I dislike the new dark theme. The contrast is too low compared with what I am used to. Could you add the old dark theme back please?

**Comment by PrimitiveWizard** [comment]
Yeah, I came here to ask the same thing. Older dark theme was a lot easier on the eyes and blended better with my browser theme.

**Comment by birtles** [comment]
Thanks for the feedback! That's good to know. Let me try tweaking it tomorrow.

Thanks again!

**Comment by birtles** [comment]
Prior to changing the colours:

![image](https://user-images.githubusercontent.com/1232595/126736430-432498ab-bb87-426b-a9fd-b2e76c635033.png)

After changing the colours:

![image](https://user-images.githubusercontent.com/1232595/126736461-149baf90-8acc-4271-890c-d2de32fcca6b.png)

Proposed update:

![image](https://user-images.githubusercontent.com/1232595/126736490-99d06ce6-da55-486f-b3b4-b97a8c0cd982.png)

**Comment by birtles** [comment]
For me the update is a little too "neon" but I don't regularly use dark mode so I'm not sure. It looks like dark mode users tend to prefer very high contrast so maybe it's ok?

**Comment by birtles** [comment]
I'm going to go ahead and release this for now. We can tweak it again if there is more feedback.

**Comment by birtles** [comment]
This is now released as version 1.1.4.

I've only released on Firefox for now simply because it's faster and easier than the other platforms. I'll wait to address some of the other regressions before re-releasing on the other platforms.

**Comment by birtles** [comment]
I'm closing this issue but please feel free to leave feedback here, positive or negative. I'm more than happy to re-open this.

**Comment by afgks** [comment]
Hello, I used photoshop to adjust the colour of background to #141211, entries to #6f6ff6 #4e4e8b, readings to #69c86b #467846, notes (abbrev, proverb, idiomatic, archaism etc) to #474186.
I think this looks much better:
![10t](https://user-images.githubusercontent.com/11691936/126765443-daef6bd9-73a0-40b3-820a-827b9e7f1b2b.png)

**Comment by birtles** [comment]
> Hello, I used photoshop to adjust the colour of background to #141211, entries to #6f6ff6 #4e4e8b, readings to #69c86b #467846, notes (abbrev, proverb, idiomatic, archaism etc) to #474186.

Thank you so much for doing this and even for providing the specific hex codes!

I'm curious about the purple colour. I've gone with blue because it is one of the 10ten brand colours and it is also the highlight colour used in Firefox's dark theme:

![image](https://user-images.githubusercontent.com/1232595/126858398-9751671b-68b7-4ace-95b3-9150e1ff1902.png)

Is there a reason to prefer purple?

I've also gone with `#1d1a19` for the black since that is also one of the 10ten brand colours although it's not overly different to `#141211`.

I agree the part-of-speech label colours could use some tweaking however.

**Comment by afgks** [comment]
> 
> 
> > Hello, I used photoshop to adjust the colour of background to #141211, entries to #6f6ff6 #4e4e8b, readings to #69c86b #467846, notes (abbrev, proverb, idiomatic, archaism etc) to #474186.
> 
> Thank you so much for doing this and even for providing the specific hex codes!
> 
> I'm curious about the purple colour. I've gone with blue because it is one of the 10ten brand colours and it is also the highlight colour used in Firefox's dark theme:
> 
> ![image](https://user-images.githubusercontent.com/1232595/126858398-9751671b-68b7-4ace-95b3-9150e1ff1902.png)
> 
> Is there a reason to prefer purple?
> 
> I've also gone with `#1d1a19` for the black since that is also one of the 10ten brand colours although it's not overly different to `#141211`.
> 
> I agree the part-of-speech label colours could use some tweaking however.

Haha, there's no particular reason other than it's closer to the older colour scheme which I am used to. I didn't know you were going for a new look for the rebrand.
Since you said you prefer blue, I picked `#6d94bc` `#33719a` for the entries, `#3c5269` for the highlighted box. Is it better?

![10t blue](https://user-images.githubusercontent.com/11691936/126866604-dde71bd8-a6a6-4cbd-99b7-3d4eb6cf9df0.png)

I wonder if you could let us customise the colours in settings if it isn't too much work. I am used to the purple colour scheme and `#1d1a19` is still a bit too light for my taste.

**Comment by birtles** [comment]
> Haha, there's no particular reason other than it's closer to the older colour scheme which I am used to. I didn't know you were going for a new look for the rebrand.
> Since you said you prefer blue, I picked `#6d94bc` `#33719a` for the entries, `#3c5269` for the highlighted box. Is it better?

Thank you once again! Sorry for my delay in responding. I quite like those colours. I wonder if anyone else on this thread has a preference of the more subdued blue than the current light blue?

> I wonder if you could let us customise the colours in settings if it isn't too much work. I am used to the purple colour scheme and `#1d1a19` is still a bit too light for my taste.

Just to check are you referring to `#1d1a19` as used in the latest version (1.1.4)? It should be almost indistinguishable from `#141211` as per your screenshot.

I'd rather not make the colours customisable if we can help it since it makes maintaining the add-on harder (e.g. if we need to add a new highlight of some kind we either need to introduce new values to customise or if we try re-using existing colours we end up with the possibility of a user's custom colours making the content unreadable).

**Comment by Tomalak** [comment]
I'm in favor of the toned-down blue. (GitHub needs a voting system)

**Comment by birtles** [comment]
I went to switch to the toned-down blue (`#6d94bc`) today but on my screen it's quite hard to read. It looks great in the screenshots above but the particular dpi/font settings on this machine means the strokes are thinner and the toned-down blue is quite hard to read.

I took that blue (`#6d94bc`) and the current blue (`#11bbff`) and put them in Eric Meyer's colour blender to get the midpoint between them: `#3fa8de` (https://meyerweb.com/eric/tools/color-blend/#6D94BC:11BBFF:1:hex). That for me is just barely readable without strain.

I tried going lower, e.g. 1/3 if the distance between the toned-down blue and the current blue, but I have to strain my eyes to read it.

I hope you don't mind if I go with `#3fa8de` for now. Obviously we can continue to tweak it as we go.
